scraper.rb

```diff
@@ -6,6 +6,13 @@ def scrape()
 
   agent = Mechanize.new
 
+  if ENV["MORPH_AUSTRALIAN_PROXY"]
```

Comment:
This change (or something similar) is already being introduced by another PR https://github.com/planningalerts-scrapers/sorell/pull/3